### ISSUE TITLE
Write new example applications

### ISSUE THREAD

**Issue by FilippoOggionni** [tudat-team/tudatpy-examples#3]
We can use this thread to come up with ideas.

From my side: e.g., simulation of two objects in LEO with high collision risks (conjunction messages). We can reproduce situations that actually happened (real collisions or avoided ones).

**Comment by gaffarelj** [comment]
Your idea is very nice, and I would suggest having more than three objects, to showcase and test the new dependent variables based on multiple bodies (cfr [this issue](https://github.com/tudat-team/tudat/issues/36)).
This could serve as a baseline for research on collision avoidance, or constellation design.

**Comment by FilippoOggionni** [comment]
Great idea! I did not see the issue you linked, but it would be a good addition indeed

**Comment by jo11he** [comment]
I think that Luana Capus, MSc student from the slack workspace, is using tudatpy for reconstructing various scenarios around the Cosmos - Iridium collision, so she may be able to provide a great starting point for an example :)

**Comment by gaffarelj** [comment]
Good idea, I will contact her. She will probably also have some input on dependent variables for multiple bodies.
